src/sidebar/combat-tracker.js

```diff
@@ -112,16 +112,6 @@ export default class YearZeroCombatTracker extends foundry.applications.sidebar.
   _getEntryContextOptions() {
     const contextMenu = super._getEntryContextOptions();
 
-    // The combat tracker will initialize context menus regardless of there being a combat active
-    if (!game.combat) return contextMenu;
-
-    // Scoped Constants
-    /** @type {YearZeroCombat} */
-    const combat = game.combat;
-
-    /** @type {Collection.<YearZeroCombatant>} EmbeddedCollection */
-    const combatants = combat.combatants;
-
```

Comment:
Why removing the scoped constants? The type definition for `YearZeroComnbat` and `YearZeroCombatant` helps the VS Code's intellisense to retrieve our custom methods added to these classes.

Reply:
Ok I understand the importance to remove lines 115-116.

But why removing lines 118-123 ?